### PR TITLE
*[iOS] bugfix:fix slider crash on iOS 8.x .

### PR DESCRIPTION
**Pull request by bluebird78999**

### CODE REVIEW

File: ios/sdk/WeexSDK/Sources/Component/WXSliderComponent.m [hunk 1]
```diff
@@ -284,6 +284,16 @@ - (BOOL)_isItemViewVisiable:(UIView *)itemView
 
 - (void)scrollViewDidScroll:(UIScrollView *)scrollView
 {
+    [NSObject cancelPreviousPerformRequestsWithTarget:self];
+    __weak typeof(self) weakSelf = self;
+    if (!weakSelf)
+    {
+        //NSLog(@"self被销毁");
```
Comment (cxfeng1) [inline ios/sdk/WeexSDK/Sources/Component/WXSliderComponent.m]: don't use Chinese comments.

**Comment by cxfeng1** [comment]
please resolve the conflicts.

File: ios/sdk/WeexSDK/Sources/Component/WXSliderComponent.m (hunk 1 above)
Comment (bluebird78999) [inline ios/sdk/WeexSDK/Sources/Component/WXSliderComponent.m]: removed

**Comment by acton393** [comment]
hi  @bluebird78999  maybe you can close this invalid pull request

**Pull request closed without merging**